main.go

```diff
@@ -210,6 +229,7 @@ CREATE TABLE IF NOT EXISTS auth(username TEXT, password TEXT);`)
 	Handler.HandleFunc("/", index)
 	Handler.HandleFunc("/play", play)
 	Handler.HandleFunc("/delete", del)
+	Handler.HandleFunc("/purge", purge)
```

Comment:
I want to save the "purge" endpoint for if / when we need to introduce more user-based moderation into the bot. I think `deleteprev` works for now, but I might even roll it into an option of `delete`  in the redesign.